### PR TITLE
LPS-168615 | Master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
https://issues.liferay.com/browse/LPS-168615

### CODE REVIEW

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
After analysis, all failures are not related to my PR.

File: modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsWCExportImport.testcase [hunk 1]
```diff
@@ -337,6 +337,62 @@ definition {
...
+	test CanImportTranslationWithEmeddedTags {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		WebContentNavigator.gotoAddCP();
+
+		PortletEntry.inputTitle(title = "WebContentTitle");
+
+		CKEditor.addSourceContent(content = "<h1>Webcontent</h1>");
+
+		PortletEntry.publish();
```
Comment (austinchiang) [inline modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsWCExportImport.testcase:351]: Use WebContent.addCP

File: modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro [hunk 2]
```diff
@@ -70,6 +70,23 @@ definition {
...
 
+	macro copyTranslateFile {
+		if (contains(${translationZipFileName}, " ")) {
+			var translationZipFileName = StringUtil.replace(${translationZipFileName}, " ", "*");
+		}
+
+		if (isSet(dependencyfile)) {
+			AntCommand(
+				locator1 = "build-test-translation.xml",
+				value1 = "copy-translation-file -Ddependencyfile=${dependencyfile} -Dtranslation.file.name=${translationFileName} -Dtranslation.zip.file.name=${translationZipFileName}");
```
Comment (austinchiang) [inline modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro:81]: Use camel case (dependencyFile)

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please fix the minor things and resend

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
Hi @austinchiang, I have updated all minor things above. Please take a look. Thank you.

**Comment by austinchiang** [comment]
LGTM

**Comment by austinchiang** [comment]
ci:forward

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**